src/EA.Weee.Web/EA.Weee.Web.csproj

```diff
@@ -2144,10 +2156,6 @@
       <Project>{A987FFF6-5386-419E-AF0A-E6C74E3323CF}</Project>
       <Name>EA.Weee.Domain</Name>
     </ProjectReference>
-    <ProjectReference Include="..\EA.Weee.RequestHandlers\EA.Weee.RequestHandlers.csproj">
```

Comment:
Should this be removed?  Is it not used by other classes in here

Reply:
It looks to have been added accidently t some point.